src/engine/Xpattern_match_aliengrep.ml

```diff
@@ -6,7 +6,7 @@ let convert_pos ~file (loc : Aliengrep.Match.loc) =
   (* single "token" spanning the whole match *)
   let charpos = loc.start in
   let line, column = Xpattern_matcher.line_col_of_charpos file charpos in
-  { Tok.str = loc.substring; pos = { charpos; file; line; column } }
+  { Tok.str = loc.substring; trace = []; pos = { charpos; file; line; column } }
```

Comment:
this can be replace with
let pos = { charpos; file; line; column} (or  let pos = mk_pos ~charpos ~file ~line ~column) in
Tok.mk_loc ~str ~pos (and use a default value for ?trace=[])